src/HttpServer.ts

```diff
@@ -36,7 +36,11 @@ export class HttpServer {
     start(): void {
         if (this.serverInstance === undefined) {
             this.serverInstance = this.app.listen(this.port, () => {
-                log.info('Listening on port', this.port);
+                // We need to be sure that this line is always printed
+                // because the tests wait for it to be logged.
+                const args = ['Listening on port', this.port];
+                console.log(...args);
```

Comment:
unnecessary console log

Reply:
Maybe the comment isn't clear, but we need this console log in the integration tests because the tests wait for this line to understand if the server is ready (please have a look at [this check](https://github.com/rsksmart/rif-relay/blob/PP-271/fix-integration-tests/test/TestUtils.ts#L169)).
If we come up with a different strategy, we could schedule the change, but for now, I'd leave this mechanism.

Reply:
I just checked the check mentioned. We need to find another way for this to work. For now, it works.